src/components/views/PlayWhites.js

```diff
@@ -9,35 +9,26 @@ import PropTypes from "prop-types";
 for playing white cards on a black Card
  */
 const PlayWhites = () => {
-  // use react-router-dom's hook to access the history
-  const[blackCard, setBlackCard] = useState(null)
-  const[gameId, setGameId] = useState(null)
-  const [cards, setCards] = useState(null)
-    const [count, setCount] = useState(0)
+   // use react-router-dom's hook to access the history
+  const[blackCard, setBlackCard] = useState(0)
```

Comment:
Sorry, only meant useState(0) for gameId, not for the card objects, for the objects you can use null (or maybe {}?) 🥲